### DISCUSSION TITLE
Text Te Speech (Streaming)  With Timestamps

### DISCUSSION THREAD

**Discussion by hjbaard** [RageAgainstThePixel/com.rest.elevenlabs#87] [Q&A]
In the api-reference of elevelabs there are 2 post methods I can't do with this client:
- Text To Speech With Timestamps https://elevenlabs.io/docs/api-reference/text-to-speech-with-timestamps
- Text To Speech Streaming With Timestamps https://elevenlabs.io/docs/api-reference/streaming-with-timestamps

Will this be added? And when could we expect that?

**Comment by StephenHodgson** [comment]
Hi @hjbaard I'll try to make some time to update this library soon. 

I've been wanting to add websocket support so hopefully in the next few weeks.

**Reply by hjbaard** [reply]
Thanks!
